poker-pals/src/components/Poker.vue

```diff
@@ -1,14 +1,805 @@
+                            v-bind:timer="playerTimer.player2"
+                        />
+                        <PlayerSeat id="playerSeat3"
+                            v-bind:classes="{betBox: 'betBoxA', youTag: 'youTagA', playerCards: 'playerCardsA'}"
+                            v-bind:dealerStatus="dealerStatuses.player3" 
+                            v-bind:_id="3" 
+                            v-bind:cards="playerCards.player3" 
+                            v-bind:cardReveal="cardReveal"
+                            v-bind:betValue="playerBets.player3"
+                            v-bind:tagName="playerTagNames.player3"
+                            v-bind:tagImage="playerTagImages.player3"
+                            v-bind:chipTotal="playerChipTotals.player3"
+                            v-bind:action="playerActions.player3"
+                            v-bind:youTag="playerYouTag.player3"
+                            v-bind:timer="playerTimer.player3"
+                        />
+                    </div>
+                    <div class="players456">
+                        <PlayerSeat id="playerSeat4"
+                            v-bind:classes="{betBox: 'betBoxB', youTag: 'youTagB', playerCards: 'playerCardsB'}"
+                            v-bind:dealerStatus="dealerStatuses.player4" 
+                            v-bind:_id="4" 
+                            v-bind:cards="playerCards.player4"
+                            v-bind:cardReveal="cardReveal"
+                            v-bind:betValue="playerBets.player4"
+                            v-bind:tagName="playerTagNames.player4"
+                            v-bind:tagImage="playerTagImages.player4"
+                            v-bind:chipTotal="playerChipTotals.player4"
+                            v-bind:action="playerActions.player4"
+                            v-bind:youTag="playerYouTag.player4"
+                            v-bind:timer="playerTimer.player4"
+                        />
+                        <PlayerSeat id="playerSeat5"
+                            v-bind:classes="{betBox: 'betBoxB', youTag: 'youTagB', playerCards: 'playerCardsB'}"
+                            v-bind:dealerStatus="dealerStatuses.player5" 
+                            v-bind:_id="5" 
+                            v-bind:cards="playerCards.player5"
+                            v-bind:cardReveal="cardReveal"
+                            v-bind:betValue="playerBets.player5"
+                            v-bind:tagName="playerTagNames.player5"
+                            v-bind:tagImage="playerTagImages.player5"
+                            v-bind:chipTotal="playerChipTotals.player5"
+                            v-bind:action="playerActions.player5"
+                            v-bind:youTag="playerYouTag.player5"
+                            v-bind:timer="playerTimer.player5"
+                        />
+                        <PlayerSeat id="playerSeat6"
```

Comment:
Currently, you're copy/pasting a ton of PlayerSeat objects. Could you maybe construct them programatically? This will also make it easier to integrate with the backend